get.sh

```diff
@@ -160,6 +160,33 @@ parseCommandLineArgs()
 	echo "TESTDIR: $TESTDIR"
 }
 
+# If the current directory contains only a single directory then squash that directory into the current directory
+squashSingleFolderContentsToCurrentDir()
+{
+        if [[ `ls -d */ | wc -l` -eq 1 ]] && [[ `find . -maxdepth 1 -mindepth 1 | wc -l` -eq 1 ]]; then
```

Comment:
These options to `find` are GNU specific so won't work on e.g. AIX/Solaris. I think this is currently set up to run on all platforms so I would expect it to fail on at least those two.

Reply:
good point :-)

Reply:
Changed to check current directory contains 1 directory and ONLY 1 directory..
Will run cross platform test and see if works, thanks

Reply:
all pass now https://github.com/adoptium/aqa-tests/pull/5750#issuecomment-2483268215